assets/js/components/billing/DataCreditsIndex.jsx

```diff
@@ -158,7 +158,13 @@ class DataCreditsIndex extends Component {
               title="Remaining Data Credits"
               bodyStyle={{ height: 90, padding: 0 }}
               extra={
-                <Link to="#" onClick={e => e.preventDefault()}>
+                <Link
+                  to="#"
+                  onClick={e => {
+                    e.preventDefault()
+                    context.toggleSurveyNotification()
+                  }}
+                >
                   <Text style={styles.tipText}>Claim More</Text>
                 </Link>
```

Comment:
it should show this conditionally, right? if I already completed all the actions and got the rest of the 10K, I shouldn't still see "Claim More"